client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
-			description:
+			title: translate( 'Two-Step Authentication' ),
+			description: translate(
 				'Your WordPress.com site is your home on the internet, and you want to keep that home safe. ' +
-				'Learn how to add an additional "lock" to your account!',
+					'Learn how to add an additional "lock" to your account!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/account-recovery/',
 			post_id: 46365,
-			title: 'Account Recovery',
-			description:
-				'At some point, you may run into a situation in which you’ve lost access to your account. Learn how to get back on track!',
+			title: translate( 'Account Recovery' ),
+			description: translate(
+				'At some point, you may run into a situation in which you’ve lost access to your account. Learn how to get back on track!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/passwords/',
 			post_id: 89,
-			title: 'Passwords And How To Use Them',
-			description:
-				'Passwords are very important to user accounts, and there may come a time when you need to change your password.',
+			title: translate( 'Passwords And How To Use Them' ),
+			description: translate(
+				'Passwords are very important to user accounts, and there may come a time when you need to change your password.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/third-party-applications/',
 			post_id: 17288,
-			title: 'Third Party Applications',
-			description:
+			title: translate( 'Third Party Applications' ),
+			description: translate(
 				'WordPress.com allows you to connect with third-party applications that ' +
-				'extend your WordPress.com site in new and cool ways.',
+					'extend your WordPress.com site in new and cool ways.'
+			),
 		},
 	],
 	purchases: [
 		{
 			link: 'http://en.support.wordpress.com/manage-purchases/',
 			post_id: 111349,
-			title: 'Manage Purchases',
-			description:
-				'Have a question or need to change something about a purchase you have made? Learn how.',
+			title: translate( 'Manage Purchases' ),
+			description: translate(
+				'Have a question or need to change something about a purchase you have made? Learn how.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/auto-renewal/',
 			post_id: 110924,
-			title: 'Subscriptions for Plans and Domains',
-			description:
+			title: translate( 'Subscriptions for Plans and Domains' ),
+			description: translate(
 				'Your WordPress.com plans and any domains you add to your sites are based ' +
-				'on a yearly subscription that renews automatically.',
+					'on a yearly subscription that renews automatically.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/discover-the-wordpress-com-plans/',
 			post_id: 140323,
-			title: 'Explore the WordPress.com Plans',
-			description:
-				"Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals.",
+			title: translate( 'Explore the WordPress.com Plans' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Explore WordPress.com Plans' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).